### ISSUE TITLE
getBoundingBox

### ISSUE THREAD

**Issue by ColichevAV** [gkjohnson/three-mesh-bvh#536]
Hi gkjohnson, I tried to get a bounding box using getBoundingBox function, but noticed that the results are different from the three js results for this particular case. Is this a bug or an intentional feature? Thanks for your work

```javascript
const vectors = [
     new Vector3(-3.8008997440338135, 0, 5.571000099182129)
    ,new Vector3(-3.8008997440338135, 0.20000001788139343, 5.571000099182129)
    ,new Vector3(-1.0976998805999756, 0.20000001788139343, 7.463799476623535)
    ,new Vector3(-1.0976998805999756, 0.20000001788139343, 7.463799476623535)
    ,new Vector3(-1.0976998805999756, 0, 7.463799476623535)
    ,new Vector3(-3.8008997440338135, 0, 5.571000099182129)
    ,new Vector3(-1.2123997211456299, 0.20000001788139343, 7.627599716186523)
    ,new Vector3(-3.915599822998047, 0.20000001788139343, 5.734799861907959)
    ,new Vector3(-3.915599822998047, 0, 5.734799861907959)
    ,new Vector3(-3.915599822998047, 0, 5.734799861907959)
    ,new Vector3(-1.2123997211456299, 0, 7.627599716186523)
    ,new Vector3(-1.2123997211456299, 0.20000001788139343, 7.627599716186523)
    ,new Vector3(-1.0976998805999756, 0, 7.463799476623535)
    ,new Vector3(-1.2123997211456299, 0, 7.627599716186523)
    ,new Vector3(-3.915599822998047, 0, 5.734799861907959)
    ,new Vector3(-3.915599822998047, 0, 5.734799861907959)
    ,new Vector3(-3.8008997440338135, 0, 5.571000099182129)
    ,new Vector3(-1.0976998805999756, 0, 7.463799476623535)
    ,new Vector3(-3.8008997440338135, 0, 5.571000099182129)
    ,new Vector3(-3.915599822998047, 0, 5.734799861907959)
    ,new Vector3(-3.915599822998047, 0.20000001788139343, 5.734799861907959)
    ,new Vector3(-3.915599822998047, 0.20000001788139343, 5.734799861907959)
    ,new Vector3(-3.8008997440338135, 0.20000001788139343, 5.571000099182129)
    ,new Vector3(-3.8008997440338135, 0, 5.571000099182129)
    ,new Vector3(-3.8008997440338135, 0.20000001788139343, 5.571000099182129)
    ,new Vector3(-3.915599822998047, 0.20000001788139343, 5.734799861907959)
    ,new Vector3(-1.2123997211456299, 0.20000001788139343, 7.627599716186523)
    ,new Vector3(-1.2123997211456299, 0.20000001788139343, 7.627599716186523)
    ,new Vector3(-1.0976998805999756, 0.20000001788139343, 7.463799476623535)
    ,new Vector3(-3.8008997440338135, 0.20000001788139343, 5.571000099182129)
    ,new Vector3(-1.0976998805999756, 0.20000001788139343, 7.463799476623535)
    ,new Vector3(-1.2123997211456299, 0.20000001788139343, 7.627599716186523)
    ,new Vector3(-1.2123997211456299, 0, 7.627599716186523)
    ,new Vector3(-1.2123997211456299, 0, 7.627599716186523)
    ,new Vector3(-1.0976998805999756, 0, 7.463799476623535)
    ,new Vector3(-1.0976998805999756, 0.20000001788139343, 7.463799476623535)
]
const geometry = new THREE.BufferGeometry().setFromPoints(vectors)
geometry.computeBoundsTree()
geometry.computeBoundingBox()
const threeBox = geometry.boundingBox
const threeMeshBvhBox = geometry.boundsTree?.getBoundingBox(new Box3())
expect(threeBox).toEqual(threeMeshBvhBox)
``` 
![image](https://github.com/gkjohnson/three-mesh-bvh/assets/54205314/838b72e5-24b4-44d9-86c8-4a43ebcf6ca3)

**Comment by gkjohnson** [comment]
Thanks for the catch - looks like the root bounding box for the BVH always includes the origin for some reason. I'll look into it when I get a chance:

<img width="480" alt="image" src="https://github.com/gkjohnson/three-mesh-bvh/assets/734200/4f3f3d26-1581-4b17-bbc3-91a154de816f">

**Comment by ColichevAV** [comment]
Maybe something like this needs to be added:
```javascript
    const fullBounds = new Float32Array(6);
    fullBounds[0] = Infinity
    fullBounds[1] = Infinity
    fullBounds[2] = Infinity
    fullBounds[3] = -Infinity
    fullBounds[4] = -Infinity
    fullBounds[5] = -Infinity

``` 
https://github.com/gkjohnson/three-mesh-bvh/blob/42c484087a40314a99ee057e9836dd6aff889f32/src/core/buildFunctions.js#L738-L742

**Comment by gkjohnson** [comment]
Just saw this - thanks for digging into it, as well. Just got it fixed yesterday and published this morning in v0.6.1
